demos/hosp_only_ww_model/wwinference.Rmd

```diff
@@ -280,7 +280,7 @@ modified with any discrete daily PMF.
 
 ```{r set-delay-distributions}
 generation_interval <- wwinference::default_covid_gi
-inf_to_hosp <- wwinference::default_covid_inf_to_hosp
+inf_to_ed <- wwinference::default_covid_inf_to_ed
```

Comment:
`wwinferrence::default_covid_inf_to_ed` doesn't exist. Just use inf to hosp and explain why, then later update this tutorial to use PyrenewH

Reply:
Resolved in https://github.com/CDCgov/pyrenew-hew/pull/186/commits/fb8e504de519bf00acabb2acd56b12d87bf38d57

Reply:
Would have been caught if we had tutorials run in CI :(

Reply:
Created #191